src/sys/prctl.rs

```diff
@@ -213,3 +215,10 @@ pub fn set_thp_disable(flag: bool) -> Result<()> {
 pub fn get_thp_disable() -> Result<bool> {
     prctl_get_bool(libc::PR_GET_THP_DISABLE)
 }
+
+/// Set an identifier (or reset it) to the address memory range.
+pub fn set_addr_name(addr: NonNull<c_void>, length: NonZeroUsize, name: &CStr) -> Result<()> {
+    let res = unsafe { libc::prctl(libc::PR_SET_VMA, libc::PR_SET_VMA_ANON_NAME, addr.as_ptr(), length, name.as_ptr()) };
```

Comment:
From the manual, to reset the name, the arg5 has to be NULL:

>  If arg5 is NULL, the name of the appropriate anonymous virtual memory areas will be reset.

Maybe we should change the `name` argument to `Option<&CStr>`?